torchao/quantization/quantize_/workflows/int4/int4_preshuffled_tensor.py

```diff
@@ -265,15 +199,16 @@ def _(func, types, args, kwargs):
     orig_input_size = input_tensor.size()
     orig_out_features = weight_tensor.shape[-2]
 
-    wq = weight_tensor._data.contiguous()
+    wq = weight_tensor.qdata.contiguous()
     group_scale = weight_tensor.group_scale.contiguous()
-    # bf16 activation
     if weight_tensor.group_zero is not None:
+        # bf16 activation
         group_zero = weight_tensor.group_zero.contiguous()
         res = torch.ops.fbgemm.bf16i4bf16_shuffled(
             input_tensor, wq, group_scale, group_zero
         )
     else:
+        # dynamically quantizes activation to fp8
         assert weight_tensor.row_scale is not None
         row_scale = weight_tensor.row_scale.contiguous()
         xq, x_scale = quantize_fp8_row(input_tensor)
```

Comment:
btw @vkuzo, we didn't use `_choose_quant_func_and_quantize_tensor` and float8 quant args yet, but we can add that if there are multiple choices of float8 activation quant in the future. please let me know if you have different thoughts here.